frontend-react/Dockerfile_dev

```diff
@@ -0,0 +1,20 @@
+FROM node:14-alpine
```

Comment:
This builds and runs ok for me locally, but I'm curious why you didn't just replace the `Dockerfile` in this directory? 

Also, naming convention-wise it looks like we use in `prime-router` a `.` vs a `_` in the filename, so if we keep this let's call it `Dockerfile.dev`.

Reply:
The old `frontend/` directory used a `Dockerfile_dev` for ./cleanslate and docker-compose. I'm not aware of the impact vs the native Dockerfile, so I named with `_dev` to match what it was before. If there is no fear with replacing `Dockerfile` and NOT use `_dev`, then sure I can replace it